### PR TITLE
Intel(R) Optimized Caffe - Add useBlacklist field

### PR DESCRIPTION
**Pull request by congxu1987**
Submitted by Cong Xu from Intel Corporation.

Signed-off-by: Cong Xu <congxu@localhost.localdomain>

### CODE REVIEW

File: ImageNet/inference/Intel_resnet50_c59xlarge_intelcaffe.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "cost": 2.34e-06,
     "top5Accuracy": 93.07,
     "timestamp": "2018-4-18",
+    "usedBlacklist": "true",
```
Comment (codyaustun) [inline ImageNet/inference/Intel_resnet50_c59xlarge_intelcaffe.json]: Please remove the quotes around `true`

File: ImageNet/inference/Intel_resnet50_c5_18xlarge_intelcaffe_latency.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
     "latency": 9.96,
     "top5Accuracy": 93.07,
     "timestamp": "2018-4-20",
+    "usedBlacklist": "true",
```
Comment (codyaustun) [inline ImageNet/inference/Intel_resnet50_c5_18xlarge_intelcaffe_latency.json]: same as above

File: ImageNet/inference/Intel_resnet50_c5_2xlarge_intelcaffe_cost.json [hunk 3]
```diff
@@ -10,6 +10,7 @@
     "cost": 1.64e-06,
     "top5Accuracy": 93.07,
     "timestamp": "2018-4-20",
+    "usedBlacklist": "true",
```
Comment (codyaustun) [inline ImageNet/inference/Intel_resnet50_c5_2xlarge_intelcaffe_cost.json]: same as above

File: ImageNet/train/Intel_Resnet50_4K_ec2_Xeon_8124M_intelcaffe.json [hunk 4]
```diff
@@ -8,6 +8,7 @@
     "hardware": "128 nodes with Xeon Platinum 8124M / 144 GB / 36 Cores (Amazon EC2 [c5.18xlarge])",
     "timestamp": "2018-04-20",
     "logFilename": "https://github.com/intel/caffe/tree/DAWN_Bench/models/intel_optimized_models/multinode/resnet_50_128_nodes_4k_batch/outputCluster-skx-128.txt.tgz",
+    "usedBlacklist": "true",
```
Comment (codyaustun) [inline ImageNet/train/Intel_Resnet50_4K_ec2_Xeon_8124M_intelcaffe.json]: same as above

File: ImageNet/train/Intel_Resnet50_ec2_Xeon_8124M_intelcaffe.json [hunk 5]
```diff
@@ -8,6 +8,7 @@
     "hardware": "64 nodes with Xeon Platinum 8124M / 144 GB / 36 Cores (Amazon EC2 [c5.18xlarge])",
     "timestamp": "2018-04-18",
     "logFilename": "https://github.com/intel/caffe/tree/DAWN_Bench/models/intel_optimized_models/multinode/resnet_50_64_nodes_2k_batch/outputCluster-skx-64.txt.tgz",
+    "usedBlacklist": "true",
```
Comment (codyaustun) [inline ImageNet/train/Intel_Resnet50_ec2_Xeon_8124M_intelcaffe.json]: same as above

File: ImageNet/train/Intel_Resnet56_ec2_Xeon_8124M_intercaffe.json [hunk 6]
```diff
@@ -8,6 +8,7 @@
     "hardware": "128 nodes with Xeon Platinum 8124M / 144 GB / 36 Cores (Amazon EC2 [c5.18xlarge])",
     "timestamp": "2018-04-19",
     "logFilename": "https://github.com/intel/caffe/tree/DAWN_Bench/models/intel_optimized_models/multinode/resnet_56_128_nodes_4k_batch/outputCluster-skx-128.txt.tgz",
+    "usedBlacklist": "true",
```
Comment (codyaustun) [inline ImageNet/train/Intel_Resnet56_ec2_Xeon_8124M_intercaffe.json]: same as above

**Review by codyaustun — CHANGES_REQUESTED** [review]
Minor correction: `"true"` is a string of the word true not the boolean value `true`. To avoid errors, please update to use the boolean value.

**Comment by congxu1987** [comment]
Sure, we have already updated our json files.

**Review by codyaustun — APPROVED** [review]

**Comment by codyaustun** [comment]
Looks good. Thanks! I'll merge this in now, but I won't change the website until all the submissions are updated.

**Comment by congxu1987** [comment]
Thanks a lot!

**Pull request merged**